src/PendingCalls/TestCall.php

```diff
@@ -223,6 +223,30 @@ public function skipOnLinux(): self
         return $this->skipOn('Linux', 'This test is skipped on [Linux].');
     }
 
+    /**
+     * Runs the current test only if the given test is running on Windows.
+     */
+    public function onlyOnWindows(): self
+    {
+        return $this->skipOnLinux()->skipOnMac();
+    }
+
+    /**
+     * Runs the current test only if the given test is running on Mac.
+     */
+    public function onlyOnMac(): self
+    {
+        return $this->skipOnWindows()->skipOnLinux();
+    }
+
+    /**
+     * Run the current test only if the given test is running on Linux.
+     */
+    public function onlyOnLinux(): self
+    {
+        return $this->skipOnWindows()->skipOnMac();
+    }
```

Comment:
```suggestion
    /**
     * Runs the current test only if the given test is running on Windows.
     */
    public function onlyOnWindows(): self
    {
        return $this->onlyOn('Windows', 'This test is skipped unless on [Windows].');
    }

    /**
     * Runs the current test only if the given test is running on Mac.
     */
    public function onlyOnMac(): self
    {
        return $this->onlyOn('Darwin', 'This test is skipped unless on [Mac].');
    }

    /**
     * Run the current test only if the given test is running on Linux.
     */
    public function onlyOnLinux(): self
    {
        return $this->onlyOn('Linux', 'This test is skipped unless on [Linux].');
    }
    
    /**
     * Skips the current test unless it is running on the given operating system.
     */
     public function onlyOn(string $osFamily, string $message): self
     {
         return $osFamily !== PHP_OS_FAMILY
             ? $this->skip($message)
             : $this;
     }
```

Reply:
Thanks, I've updated the code based on your suggestions. The only difference is that I kept the 'onlyOn' method private.